nftest/common.py

```diff
@@ -27,6 +27,14 @@ def remove_nextflow_logs() -> None:
         else:
             os.remove(file)
 
+def resolve_single_path(path:str) -> Path:
+    """ Resolve wildcards in path and ensure only a single path is identified """
+    expanded_paths = glob.glob(path)
+    assert len(expanded_paths) == 1, \
+        f"{path} resolved to 0 or more than 1 files. Assertion failed."
```

Comment:
I suggest outputting the discovered paths so users can attempt to solve the problem.

Reply:
I concur - this is a little cryptic:

```
2023-10-24 17:01:28,738 - NFTest - ERROR - ('/hot/software/pipeline/pipeline-filter-RNAEditingSite/Nextflow/development/unreleased/main/A-mini-n2/filter-RNAEditingSite-*/CPCG0196-F1-A-mini-n2/REDItools-1.3/output/CPCG0196-F1-A-mini-n2_annotated.txt resolved to 0 or more than 1 files. Assertion failed.',)
```

Reply:
It may also be a good idea to print quotes around the path on the off chance the value is empty. For example:

```python
>>> path=''
>>> f"{path} resolved to 0 or more than 1 files. Assertion failed."
' resolved to 0 or more than 1 files. Assertion failed.'
```

as opposed to

```python
>>> f"`{path}` resolved to 0 or more than 1 files. Assertion failed."
'`` resolved to 0 or more than 1 files. Assertion failed.'
```

This is something I've seen often, so it makes sense to code defensively for it.

Reply:
Gotcha, I'll add both changes!